### DISCUSSION TITLE
v5.7.0.0

### DISCUSSION THREAD

**Discussion by stax76** [mpvnet-player/mpv.net#382] [General]
- Improved title and chapter menu for Blu-Rays.
- Fix of conf folder virtualization issue of MS Store version.
- MS Store page no longer displays ARM and x86 support.

<hr /><em>This discussion was created from the release <a href='https://github.com/stax76/mpv.net/releases/tag/v5.7.0.0'>v5.7.0.0</a>.</em>

**Comment by TheBestPessimist** [comment]
I no longer see a portable version which has all the scripts and everything. is this something expected or am i too eager and that version will be updated later?

**Reply by stax76** [reply]
The scripts were removed, but they are still available here:

https://github.com/stax76/mpv.net/tree/master/src/Scripts

It's also no longer possible to start scripts from the startup folder, scripts must be in the config folder.

I want to port the C-Sharp scripts to Lua and I want to move the Lua scripts to an independent repo, and also separate real useful scripts from pure example scripts.

When these changes happen, I probably write it to the changelog.

**Reply by TheBestPessimist** [reply]
Thanks for the answer, however i was not talking about the scripts. I was interested in having the folder `portable_config` available in the zip.

I know that one can create manually the folder, but maybe possible users don't really know that.
For me the update was smooth as i already have everything set up, but they might not know.

**Comment by bitsper2nd** [comment]
Nice one, Frank. I didn't know there was an ms store version. Now I can finally update mpv.net with winget on Windows 11.

**Comment by YuiTatsumi** [comment]
Nice but where is the executable version of v5.7.0.0? 
Just like [mpv.net-5.4.9.0-setup.exe](https://github.com/stax76/mpv.net/releases/download/5.4.9.0/mpv.net-5.4.9.0-setup.exe) 
Thank you.

**Reply by stax76** [reply]
It's no longer supported, but it's now available in the Microsoft Store (and also winget). I've submitted this version to the store, unfortunately it's not yet approved, it might take 1-2 days.

**Comment by bitsper2nd** [comment]
Thinking of making a new icon for mpvnet. What do you say, @stax76? You can see my previous work on this [image viewer](https://github.com/sylikc/jpegview/discussions/45#discussioncomment-2504320).

**Comment by zenwav** [comment]
Isn't Vulkan supported any more? If I set the gpu-api to vulkan, mpv.net doesn't start at all.

**Reply by zhongfly** [reply]
The basic settings of mpv.net are the same as those of mpv, mpv still supports vulkan, and mpv.net naturally can.
In order to better find the problem, it is better to post mpv.conf

**Reply by zenwav** [reply]
My mpv.conf just contains the following 5 lines
profile = gpu-hq
vo = gpu
gpu-api = vulkan
gpu-context = winvk
hwdec = auto

The config works as expected when I use it on mpv, but for some reason, mpv.net won't start with these configs. The last version which works correctly is [5.4.9.0]

**Reply by zhongfly** [reply]
I tested the new 5.8.0.0 version and updated mpv-2.dll to the version compiled on May 7, and compared it with the mpv.exe compiled at the same time, it is true that mpvnet cannot be started. Error message: Error opening/initializing the VO window.

**Reply by zenwav** [reply]
Is there any way to fix this issue? I would really like to use MPV with vulkan on my system as I want to use complex shaders and therefore need to remove as much overhead as possible.

**Reply by zhongfly** [reply]
I think it is a bug of mpvnet, I have opened an issue ( https://github.com/stax76/mpv.net/issues/415 ) , waiting for the developer to solve it
